src/main/java/mrriegel/storagenetwork/block/cable/BlockCable.java

```diff
@@ -180,11 +228,55 @@ public boolean onBlockActivated(World worldIn, BlockPos pos, IBlockState state,
     return false;
   }
 
+  @SuppressWarnings("deprecation")
+  @Override
+  public void onBlockClicked(World worldIn, BlockPos pos, EntityPlayer playerIn) {
+    if (!playerIn.isSneaking()) {
+      StorageNetwork.log("no sneak");
+      return;
+    }
+    TileCable tile = getTileCable(worldIn, pos);
+    if (tile == null) {
+      StorageNetwork.log("no tile");
+      return;
+    }
+    ItemStack heldStack = playerIn.getHeldItemMainhand();
+    if (heldStack == null || heldStack.isEmpty()) {
+      tile.facadeState = null;
+      worldIn.markAndNotifyBlock(pos.toImmutable(), worldIn.getChunk(pos), getDefaultState(), getDefaultState(), 1 | 2);
+      tile.markDirty();
+      StorageNetwork.log("unset facade");
+      return;
+    }
+    Block block = Block.getBlockFromItem(heldStack.getItem());
+    if (block == null || block == Blocks.AIR) {
+      StorageNetwork.log("no block");
+      return;
+    }
+    int meta = heldStack.getMetadata();
+    RayTraceResult mouseOver = Minecraft.getMinecraft().objectMouseOver;
+    float f = (float) (mouseOver.hitVec.x - (double) pos.getX());
+    float f1 = (float) (mouseOver.hitVec.y - (double) pos.getY());
+    float f2 = (float) (mouseOver.hitVec.z - (double) pos.getZ());
+    IBlockState state = block.getStateForPlacement(worldIn, pos, mouseOver.sideHit, f, f1, f2, meta, playerIn);
+    if (state == null || state.getRenderType() != EnumBlockRenderType.MODEL) {
```

Comment:
Works perfectly! Stairs metadata works exactly right and i can put at any orientation, they even connect around corners.
Glass has some issues that i will probably leave without changes, very minor. because of render type MODEL thats a sold block kind of deal, so you can see here two glass stacked vertically the inner layers dissapear on real blocks and dont for cable facades. (and dark shadow under the blue stained glass flickers) its fine illl probably just leave it like this. if players dont like the look they can use non transparent stuff for facades, just noting here for complete-ness  

![glass left vs right](https://github.com/user-attachments/assets/b37d6c86-ebeb-42f8-9e60-87eebd521184)
![517 glass transp](https://github.com/user-attachments/assets/83875206-0cfa-4127-8d79-6d273328074e)



![stair swork perfectly](https://github.com/user-attachments/assets/8738d6c9-a13a-4fca-a91b-8db5d6f4d67e)